libqtile/widget/notify.py

```diff
@@ -51,7 +51,9 @@ class Notify(base._TextBox):
     defaults = [
         ("foreground_urgent", "ff0000", "Foreground urgent priority colour"),
         ("foreground_low", "dddddd", "Foreground low priority  colour"),
-        ("default_timeout", None, "Default timeout (seconds) for notifications"),
+        ("default_timeout_low", 5, "Default timeout (seconds) for low urgency notifications."),
+        ("default_timeout", 10, "Default timeout (seconds) for normal notifications"),
```

Comment:
This means that it now has 10 seconds instead of `None` right? Could you add a change to the changelog stating this?

Reply:
That's correct. I'll put a not in the changelog.